sdks/python/apache_beam/examples/cookbook/mergecontacts_test.py

```diff
@@ -18,10 +18,10 @@
 """Test for the mergecontacts example."""
 
 import logging
-import tempfile
+import re
 import unittest
 
-from apache_beam.examples.cookbook import mergecontacts
```

Comment:
It's not much of a test of mergecontacts if mergecontacts isn't even imported. 

I suggest we leave this test as is, as reading and writing are a core part of the functionality. (It doesn't do much else.)